garage/experiment/local_tf_runner.py

```diff
@@ -43,7 +43,7 @@ class LocalRunner:
 
     """
 
-    def __init__(self, sess=None, max_cpus=1):
+    def __init__(self, sess=None, max_cpus=1, restore_from=None):
```

Comment:
how about instead there's an alternative constructor called `Runner.restore_from()`?

Reply:
That looks better.

Reply:
I found that the interface will be `Runner.restore_from(cls, snapshot_dir, sess=None, max_cpus=1)`. This is a little bit awkward because it cannot be something as simple as `Runner.restore_from(cls, snapshot_dir)`. 

If consider `runner.restore()` a counterpart of `runner.setup()`, then calling either `restore()` or `setup()` is required to set up an experiment. Following this idea, what about just removing support of `restore_from` from constructor.

Reply:
ok sgtm